modern/src/LoginPage.js

```diff
@@ -87,54 +88,78 @@ const LoginPage = () => {
-
-          <TextField
-            margin='normal'
-            required
-            fullWidth
-            error={failed}
-            label={t('userPassword')}
-            name='password'
-            value={password}
-            type='password'
-            autoComplete='current-password'
-            onChange={handlePasswordChange} />
-
-          <FormControl fullWidth margin='normal'>
-            <div className={classes.buttons}>
-              <Button type='button' variant='contained' onClick={handleRegister} disabled={!registrationEnabled}>
-                {t('loginRegister')}
-              </Button>
-              <Button type='submit' variant='contained' color='primary' disabled={!email || !password}>
+        <form className={classes.form} onSubmit={handleLogin}>
+          <Grid container direction='column' spacing={3}>
+            <Grid item>
+              {matches && <Logo fill='#333366'/>}
+            </Grid>
+            <Grid item>
+              <TextField
+                required
+                fullWidth
+                error={failed}
+                label={t('userEmail')}
+                name='email'
+                value={email}
+                autoComplete='email'
+                autoFocus
+                onChange={handleEmailChange}
+                helperText={failed && 'Invalid username or password'}
+                variant='filled' />
+            </Grid>
+            <Grid item>
+              <TextField
+                required
+                fullWidth
+                error={failed}
+                label={t('userPassword')}
+                name='password'
+                value={password}
+                type='password'
+                autoComplete='current-password'
+                onChange={handlePasswordChange}
+                variant='filled' />
+            </Grid>
+            <Grid item>
+              <Button 
+                type='submit'
+                variant='contained' 
+                color='secondary'
+                disabled={!email || !password}
+                fullWidth>
                 {t('loginLogin')}
               </Button>
-            </div>
-          </FormControl>
+            </Grid>
+            <Grid item container>
+              <Grid item>
+                <Button onClick={handleRegister} disabled={!registrationEnabled} color="secondary">
+                  {t('loginRegister')}
+                </Button>
+              </Grid>
+              <Grid item xs>
+                <FormControl variant="filled" fullWidth>
+                  <InputLabel>{t('loginLanguage')}</InputLabel>
+                  <Select>
+                    <MenuItem value="en">English</MenuItem>
+                  </Select>
+                </FormControl>
+              </Grid>
+            </Grid>
+            <Grid container justify="flex-end">
+              <Grid item>
+                <Button color="primary">
```

Comment:
Since we won't be implementing the password reset anytime soon, we can take it out. But leave the grid in place though

Reply:
We actually implemented it recently.

Reply:
We should also probably replace it with a link instead of button. Buttons use all caps and it looks a bit odd.

Reply:
Sounds good. I don't have a mockup for the password recovery yet. You can use a similar implementation as the register form

Reply:
Added  textTransformation='none' which will work for all the buttons.

Reply:
Let's not update the style for buttons. We should stay close to the material design guideline. Let's use a link instead